src/index.js

```diff
@@ -645,17 +650,10 @@ class AvatarEditor extends React.Component {
   // rect dimensions used to draw the inner rect, and the bleed marks
   getRect(bleed = {}, posX, posY, bleedDistance, canvasW, canvasH) {
     const distance = !isEmpty(bleed) ? bleedDistance : 0
```

Comment:
I'm getting a lint error on this line when I run npm i:

/Users/mikerudling/v/react-avatar-editor/src/index.js
  652:11  error  'distance' is assigned a value but never used  no-unused-vars

Reply:
Removing this line will cause a further lint error because isEmpty lodash import is then not needed.

Reply:
Yep, cleaned up the lint issues. Not sure why it didn't complain before.